.github/workflows/CI.yml

```diff
@@ -487,15 +493,20 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
-          aws-region: us-west-2
+          aws-region: ${{ env.INTEGRATION_TEST_REGION_2 }}
 
       - name: Create SSM package
```

Comment:
I thought it would be too overkill for separate releases ssm and release image in this PR and do it in another PR. However, its fine to do this in this PR.